tsconfig.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "./",
+    "declaration": true,
+    "jsx": "react",
+    "module": "ES6",
+    "moduleResolution": "node",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "outDir": "dist",
+    "paths": {},
+    "sourceMap": true,
+    "strictNullChecks": true,
+    "target": "es2017"
```

```suggestion
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "baseUrl": "./",
    "declaration": true,
    "jsx": "react",
    "module": "ES6",
    "moduleResolution": "node",
    "noFallthroughCasesInSwitch": true,
    "noImplicitReturns": true,
    "sourceMap": true,
    "strictNullChecks": true,
```

no need for `outDir` or `target` if we're running `tsc` in `no-emit` every time

also removed `paths`, I don't think it's necessary if it's just an empty object